packages/elements/src/datetime-picker/index.ts

```diff
@@ -1159,13 +1178,40 @@ export class DatetimePicker extends ControlElement implements MultiValue {
     ></ef-time-picker>`;
   }
 
+  /**
+   * Create calendar slot
+   * @param id Calendar identifier
+   * @returns calendarSlots slots that will cascade to calendar
+   */
+  private createCalendarSlots(id: string): HTMLSlotElement[] | null {
+    if (!this.opened) {
+      return null;
+    }
+
+    const querySlots = Array.from(this.querySelectorAll('[slot]'));
+    return querySlots
+      .filter((slot) => {
+        const isToSlot = id === 'calendar-to' && slot.slot.startsWith('to-');
```

Comment:
Could we move 'calendar-to' and 'calendar' into variables for better readability ?
Ex.
const CALENDAR_ID = 'calendar';
const CALENDAR_TO_ID = 'calendar-to';

Reply:
done